config/default.yaml

```diff
@@ -289,15 +289,15 @@ webhooks:
   maxBytes: 1048576 # 1MB
 
 coverage:
-  default: true
-  # plugin: sonar
-  # sonar:
-  #   sdApiUrl: https://api.screwdriver.cd
-  #   sonarHost: https://sonar.screwdriver.cd
-  #   adminToken: your-sonar-admin-token
-  #   sdUiUrl: https://cd.screwdriver.cd
-  #   sonarEnterprise: false
-  #   sonarGitAppName: "Screwdriver Sonar PR Checks"
+  default: "false"
```

Comment:
I have noticed this issue. I have made below change locally in `default.yaml` to mitigate that
```
bookends:
  # Plugins for build setup
  default:
    setup:
      - scm
      - screwdriver-cache-bookend
    teardown:
      - screwdriver-artifact-bookend
#      - screwdriver-coverage-bookend
      - screwdriver-cache-bookend
```

Reply:
Yes, I’ve done the same, but I want to ensure that when a developer boots up our dev environment, everything is fully automated, without any manual steps.